pyfstat/make_sfts.py

```diff
@@ -132,14 +131,14 @@ def _get_setup_from_noiseSFTs(self):
         self.sftfilenames = []  # This refers to the MFD output!
 
         # SWIG-LAL does not import this functionality
-        gps_to_float = lambda x: x.gpsSeconds
+        gps_to_int = lambda x: x.gpsSeconds
 
         for ifo_catalog in noise_multi_sft_catalog.data:
             ifo_name = lalpulsar.ListIFOsInCatalog(ifo_catalog).data[0]
 
             time_stamps = lalpulsar.TimestampsFromSFTCatalog(ifo_catalog)
-            this_start_time = gps_to_float(time_stamps.data[0])
-            this_end_time = gps_to_float(time_stamps.data[-1])
+            this_start_time = gps_to_int(time_stamps.data[0])
+            this_end_time = gps_to_int(time_stamps.data[-1])
```

Comment:
Here `this_end_time` is a "last SFT starttime" as per internal lalpulsar convention...